app/types/api_slot_list_type.rb

```diff
@@ -1,7 +1,5 @@
 class ApiSlotListType < ActiveModel::Type::Value
   def cast(value)
-    api_slots = value.map { |api_slot| Nomis::ApiSlot.new(api_slot) }
-
-    ApiSlotList.new(api_slots)
+    value.map { |api_slot| Nomis::ApiSlot.new(api_slot) }.dup.freeze
```

Comment:
What's rational behing the dup? #map already returns a copy of the original "value"

Reply:
Yep good point @StupidCodeFactory - typo on my part
Will remove